modules/serverless-kafka-source-kn.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_sources/serverless-kafka-source.adoc
+[id="serverless-kafka-source-kn_{context}"]
+= Creating a Kafka event source using the `kn` CLI
+
+This section describes how to create a KafkaSource using the `kn` command.
+
+:FeatureName: Creating a Kafka event source using the `kn` CLI
+include::../modules/technology-preview.adoc[leveloffset=+0]
```

Comment:
Might want to make a small Jira task to track removing the TP notes for Kafka when it goes GA?